src/app/editor.js

```diff
@@ -2,10 +2,32 @@
 
 var EventManager = require('../lib/eventManager')
 
+var csjs = require('csjs-inject')
 var ace = require('brace')
 var Range = ace.acequire('ace/range').Range
 require('../mode-solidity.js')
 
+var css = csjs`
```

Comment:
LESS, SASS, STYLUS and the likes are a seperate custom language not native to the web to add features to styling that could be added by a native web language (e.g. javascript).

One core feature that is enabled by javascript beside removing the need for transpilation is, that, if - for example - there would ever be a decision to make styling dependent on user input (e.g. fade color based on users mouse position or change styling based on user GPS or audio/webcam etc.. input, then there will be a "state explosion" of styling if that would be tried with defining different styling states, because the changes are continuous.

But if javascript would be employed here - it's easily possible to solve that problem, because javascript can change stuff at runtim, while normal css preprocessors can only work during build time.

I can recommend https://www.npmjs.com/package/csjs - in case the "LESS decision" is not set in stone. :P